misc/toml-parser/src/main/java/io/ballerina/toml/internal/parser/TomlParser.java

```diff
@@ -468,7 +488,7 @@ private STNode parseDoubleQuoteToken(ParserRuleContext ctx) {
 
     private STNode parseStringContent() {
         STToken nextToken = peek();
-        if (nextToken.kind == SyntaxKind.IDENTIFIER_LITERAL) {
+        if (nextToken.kind == SyntaxKind.IDENTIFIER_LITERAL || nextToken.kind == SyntaxKind.DOUBLE_QUOTE_TOKEN) {
```

Comment:
if the syntaxkind is DOUBLE_QUOTE_TOKEN then return STNodeFactory.createEmptyNode();
Also need to fix recovery for this method with above change

Reply:
1. if there's IDENTIFIER_LITERAL we consume it and return
2. if there's DOUBLE_QUOTE_TOKEN we simply create emptyNode and return (need to make the token optional in the Syntax API)
3. something is wrong. Hence go to recovery and re-parse

Similar case from BallerinaParser.java https://github.com/ballerina-platform/ballerina-lang/blob/7289b5b33d06787e4ebfa7159d6eab19e5545f80/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParser.java#L6055-L6066